### DISCUSSION TITLE
Typo in Documentation?

### DISCUSSION THREAD

**Discussion by ajmetz** [mojolicious/mojo#1806] [Q&A]
If I follow the advice here...
[https://docs.mojolicious.org/Mojolicious/Guides/Cookbook#Making-your-application-installable](https://docs.mojolicious.org/Mojolicious/Guides/Cookbook#Making-your-application-installable)
...to remove `Mojolicious::Command::Author` from the command namespace [setting just `Mojolicious::Commands` in the example given], I get the error...
`Unknown command "psgi", maybe you need to install it?`
...on the web app, while it's running via Plack.

Looking at what was installed in my Perlbrew, I realised there was Mojolicious::Command::psgi that was probably needed.
So, is this a typo problem in the documentation, and the example given should have set `Mojolicious::Command` rather than `Mojolicious::Commands`...?
Removing the `s` has seemed to fix the problem...

I'm new to packaging apps for installation, and was hoping to have a best of both worlds - a mojolicious app that could be ran as normal via plack, and could also be installed. The reason for packaging a Mojolicious app for installation is because I'm told packaging apps for installation is something I ought to learn how to do, ^_^. At present I just run all my webapps via plackup and use git clone if I need copies anywhere else.

When I read the guide on contributing to Mojolicious, it mentions not to do pull requests for simple typo corrections.
That's why I thought I'd post here.

**Comment by kraih** [comment]
That should probably be `Mojolicious::Command`.

**Reply by ajmetz** [reply]
Cool, =).
Will you be able to update https://docs.mojolicious.org/Mojolicious/Guides/Cookbook#Making-your-application-installable to reflect that? Or is it something I can do via github somehow? [New to using GitHub - only used Git and Git Flow at the commandline so far].

From...
```
  # Exclude author commands
  $self->commands->namespaces(['Mojolicious::Commands']);
```
To...
```
  # Exclude author commands
  $self->commands->namespaces(['Mojolicious::Command']);
```
